src/FunderMaps.WebApi/Controllers/Portal/IncidentPortalController.cs

```diff
@@ -0,0 +1,112 @@
+using AutoMapper;
+using FunderMaps.Controllers;
+using FunderMaps.Core.Entities;
+using FunderMaps.Core.UseCases;
+using FunderMaps.Helpers;
+using FunderMaps.WebApi.DataTransferObjects;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+
+#pragma warning disable CA1062 // Validate arguments of public methods
+namespace FunderMaps.WebApi.Controllers.Portal
+{
+    /// <summary>
+    ///     Endpoint controller for incident operations.
+    /// </summary>
+    [AllowAnonymous]
+    [Route("incident-portal")]
+    public class IncidentPortalController : BaseApiController
+    {
+        private readonly IMapper _mapper;
+        private readonly IncidentUseCase _incidentUseCase;
+        private readonly GeocoderUseCase _geocoderUseCase;
+
+        /// <summary>
+        ///     Create new instance.
+        /// </summary>
+        public IncidentPortalController(IMapper mapper, IncidentUseCase incidentUseCase, GeocoderUseCase geocoderUseCase)
+        {
+            _mapper = mapper ?? throw new ArgumentNullException(nameof(mapper));
+            _incidentUseCase = incidentUseCase ?? throw new ArgumentNullException(nameof(incidentUseCase));
+            _geocoderUseCase = geocoderUseCase ?? throw new ArgumentNullException(nameof(geocoderUseCase));
+        }
+
+        /// <summary>
+        ///     Upload document to the backstore.
+        /// </summary>
+        /// <param name="input">See <see cref="IFormFile"/>.</param>
```

Comment:
For referencing another file, will we use the standard to add the word "See"?

Reply:
I've always done so